src/clj/web/lobby.clj

```diff
@@ -112,29 +112,26 @@
       (if-not old (send-lobby)))))
 
 (defn player?
-  "True if the given client-id is a player in the given gameid"
-  [client-id gameid]
-  (when-let [game (game-for-id gameid)]
-    (some #(when (= client-id (:ws-id %)) %) (:players game))))
+  "True if the given client-id is a player in the given game"
+  [client-id game]
+    (some #(when (= client-id (:ws-id %)) %) (:players game)))
 
 (defn first-player?
-  "True if the given client-id is the first player in the given gameid"
-  [client-id gameid]
-  (when-let [game (game-for-id gameid)]
-    (= client-id (-> game :players first :ws-id))))
+  "True if the given client-id is the first player in the given game"
+  [client-id game]
+    (= client-id (-> game :players first :ws-id)))
 
 (defn spectator?
-  "True if the given client-id is a spectator in the given gameid"
-  [client-id gameid]
-  (when-let [game (game-for-id gameid)]
-    (not (some #(when (= client-id (:ws-id %)) %) (:players game))))) ; Faster to check players than to check spectators
+  "True if the given client-id is a spectator in the given game"
+  [client-id game]
+    (some #(when (= client-id (:ws-id %)) %) (:spectators game)))
```

Comment:
looks like some indentation stuff was left in here, along with the functions above

Reply:
ughh, yeah thanks. Indentation for functions is normal for me so I didnt even notice.